tests/unit/test_windows_compatibility.py

```
@@ -0,0 +1,262 @@
+        """Test Windows-specific process termination."""
+        from ocode_python.tools.bash_tool import _process_manager
+
+        # Mock process that needs termination
+        mock_process = AsyncMock()
+        mock_process.pid = 1234
+        mock_process.terminate.return_value = None
+        mock_process.wait.side_effect = asyncio.TimeoutError()
+        mock_process.kill.return_value = None
+
+        # Test that taskkill is called on Windows
+        await _process_manager.cleanup_process(mock_process)
```

Pylint (3.3.7)

[error] Instance of 'ProcessManager' has no 'cleanup_process' member

(E1101)